docs/docs/cmd/onenote/notebook/notebook-add.md

```diff
@@ -0,0 +1,127 @@
+# onenote notebook add
+
+Create a new OneNote notebook.
+
+## Usage
+
+```sh
+m365 onenote notebook add [options]
+```
+
+## Options
+
+`-n, --name <name>`
+: Name of the notebook. Notebook names must be unique. The name cannot contain more than 128 characters or contain the following characters: ?\*/:<>
+
+`--userId [userId]`
+: Id of the user. Use either userId or userName, but not both.
+
+`--userName [userName]`
```

Comment:
I wonder if this option name is understandable and if something like `upn` wouldn't be better. `userName` could be like the login name, or email, or display name.